README.md

```diff
@@ -200,4 +201,14 @@ NULL ...
 ### Internals
 ![](./assets/internals.png)
 
+## Release versioning
```

Comment:
```suggestion
## Versioning
```

See this [client library README](https://github.com/googleapis/nodejs-spanner/blob/831846cd86e91fdb343ac2f64a6b257877e0e157/.kokoro/presubmit/node10/system-test-against-emulator.cfg) as an example of how to explain versioning in the README.

Reply:
Have we yet decided if we are promoting this to alpha, beta etc? I thought for now we are documenting the basic version just to get a release.

Reply:
The version documentation is independent of releases. It's just a section in the docs to show what release scheme we're using. So it'll be whatever defines the <django version><pre-release><version> format.

In any case, we'll always start with alpha. I don't think we want to publish something before alpha to pypi.